src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -651,6 +651,11 @@ private static async Task<OrchestrationState> GetOrchestrationInstanceStateAsync
             string taskHubName,
             string connectionName)
         {
+            if (this.httpApiHandler == null)
+            {
+                throw new InvalidOperationException("CreateHttpManagementPayload not supported.");
```

Comment:
Consider replacing the message with something like:

"IDurableClient.CreateHttpManagementPayload is not supported for IDurableClient instances created outside of an Azure Functions application."